File: 1614. Maximum Nesting Depth of the Parentheses/solution.py
class Solution:
    def maxDepth(self, s: str) -> int:
        depth,maximum = 0,0
        for x in s:
            if (x == "("):
                depth+=1
            elif(x == ")"):
                depth-=1
            if (depth > maximum):
                    maximum = depth
        return (maximum)

File: 1614. Maximum Nesting Depth of the Parentheses/solution.java
class Solution {
    public int maxDepth(String s) {
        int length = s.length();
        var depth = 0;
        var max = 0;
        for (int i = 0; i < length; i++) {
            if (s.charAt(i) == '(') {
                depth++;
                // if(max<depth) max = depth;
            } else if (s.charAt(i) == ')') {
                depth--;
            }
            if (max < depth)
                max = depth;
        }

        return max;

    }
}